integration/testnet/network.go

```diff
@@ -866,7 +871,7 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 			net.AccessPortsByContainerName[nodeContainer.Name()] = hostGRPCPort
 			net.AccessPorts[AccessNodeAPIProxyPort] = hostHTTPProxyPort
 
-			if nodeConf.SupportsUnstakedNodes {
+			if nodeConf.SupportsObserverNodes || nodeConf.Role == flow.RoleObserverService {
```

Comment:
So, let's just clarify.
Observer bind address points to -> Access node public network port that has observer support
However, we brainstormed that in order to have a scalable observer network, we may connect observers to each other in the future, this is why it may be necessary. What do you think?

Reply:
All right. My understanding was that we will be able to scale these observer nodes as a streaming chain to each other to support large customers. If this is not the case and there is no chaining, we do not need the flag indeed. I understand the scope better now.

Am I correct that the public network key is shared, and it should be set in both the observer and the access node?

Reply:
So there is a bit of a misunderstanding here. 

> My understanding was that we will be able to scale these observer nodes as a streaming chain to each other to support large customers. 

Yes, this is correct. I'm saying we anyways don't need the flag, because for observers, there is no option to not support unstaked nodes. Does this make sense?

Whereas for Access Nodes, they can be configured to not participate in the public network.

Reply:
> Am I correct that the public network key is shared, and it should be set in both the observer and the access node?

For the integration tests, when setting up an Observer, we just remember to add the `--bootstrap-node-addresses` and `--bootstrap-node-public-keys` flags to point to the Access node that the observer will bootstrap from. 

However, the Observer also needs its own network key, it cannot just reuse the Access Node's key.